client/src/components/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,38 @@
+import { Center, Spinner as PageSpinner } from '@chakra-ui/react'
+
+interface SpinnerProps {
+  size?: 'sm' | 'md' | 'lg' | 'xl' | 'xs'
+  thickness?: string
+  emptyColor?: string
+  color?: string
+  speed?: string
+}
+
+interface PageSpinnerProps extends SpinnerProps {
+  width?: string | number
+  height: string | number
+}
+
+const Spinner = ({
+  width,
+  height,
+  size,
+  thickness,
+  emptyColor,
+  color,
+  speed,
+}: PageSpinnerProps): JSX.Element => {
+  return (
+    <Center className="spinner" h={height} w={width}>
```

Comment:
would it work if we applied the width and height on the `Spinner` itself instead? that would allow you to spread the props so the whole component can look like this:
```ts
import {
  Center,
  Spinner as ChakraSpinner,
  SpinnerProps,
} from '@chakra-ui/react'

const Spinner = (props: SpinnerProps): JSX.Element => {
  return (
    <Center>
      <ChakraSpinner {...props} />
    </Center>
  )
}

export default Spinner
```

Reply:
Looks this if we apply `width` and `height` on `Spinner`, so probably not the best.

https://user-images.githubusercontent.com/56983748/133736244-d8d9b44a-7202-488a-98e6-1dd89f8b5982.mov



Without `width` and `height` applied to `Center`:

![Screenshot 2021-09-17 at 2 40 50 PM](https://user-images.githubusercontent.com/56983748/133736499-2e2ae716-e0c1-41fa-97f8-661c74ac224c.png)

With `height` applied to `Center`:

![Screenshot 2021-09-17 at 2 41 03 PM](https://user-images.githubusercontent.com/56983748/133736596-af9e5c0d-0b6f-40ed-adfd-f853dc40f22d.png)

Reply:
Not sure if this is a dumb question, but is there a need for a custom component file for the spinner? Would it make sense for us to use the Spinner component from Chakra directly? It looks very similar based on the docs: https://chakra-ui.com/docs/feedback/spinner

If we want customisations (which are shared across the site), we could extend the theme as well: https://chakra-ui.com/docs/theming/customize-theme#customizing-component-styles

Reply:
is it just a matter of adjusting the width and height to a more appropriate size?

Reply:
unfortunately no, that changes the size of the spinner but not its positioning